### ISSUE TITLE
Template Source Manager doesn't allow URLs to github without the raw link 

### ISSUE THREAD

**Issue by groeges** [eclipse/codewind#540]
When attempting to use "Manage Template Sources" to add a new template, you are unable to add a URL to a github repository that doesn't use a raw link.  

The URL I was trying to add was the `Kabanero` URL: https://github.com/kabanero-io/collections/releases/latest/download/kabanero-index.json.

The `Appsody Stacks` template that comes as default doesn't use a raw link!!!

Unfortunately, the only reference to the kabanero-index.json is part of a Git Release, which doesnt have a raw link :-(

**Codewind version:** 0.4.0
**OS:** MacOS

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description:**

**Steps to reproduce:**
1. Ensure you have Codewind 0.4.0.
2. Right Click on "Projects" under "Codewind" in Explorer and select "Manage Template Sources"
3. In the window that is shown, put the URL https://github.com/kabanero-io/collections/releases/latest/download/kabanero-index.json into the entry box for the template.
4. The UI will display an error saying "For GitHub URLs, you must use the raw link" 

**Workaround:** 

No workaround found.

**Comment by tetchel** [comment]
🤦‍♂

**Comment by tetchel** [comment]
You can work around it by editing `codewind-workspace/.config/repository_list.json` by hand to add your new repository, and the restarting Codewind. Note the object at the bottom of the array:

```
[
  {
    "url": "https://raw.githubusercontent.com/kabanero-io/codewind-templates/master/devfiles/index.json",
    "description": "The default set of templates for new projects in Codewind.",
    "enabled": true,
    "protected": true,
    "projectStyles": [
      "Codewind"
    ],
    "name": "Default templates"
  },
  {
    "url": "https://github.com/kabanero-io/collections/releases/download/v0.1.2/kabanero-index.json",
    "description": "Kabanero Collections",
    "enabled": false,
    "projectStyles": [
      "Appsody"
    ]
  },
  {
    "description": "Appsody Stacks - appsodyhub",
    "url": "https://github.com/appsody/stacks/releases/latest/download/incubator-index.json",
    "enabled": false,
    "protected": true,
    "projectStyles": [
      "Appsody"
    ]
  },
  {
    "description": "Kabanero",
    "url": "https://github.com/kabanero-io/collections/releases/latest/download/kabanero-index.json",
    "enabled": true,
    "protected": false,
    "projectStyles": [
      "Appsody"
    ]
  }
]
```

**Comment by groeges** [comment]
@tetchel Thanks. The workaround worked.

Is it possible to use file:// URLs to specify the templates?

**Comment by tetchel** [comment]
No, http(s) only. Do you want to point to a local template, or a local template repository (`index.json`) ?

**Comment by tetchel** [comment]
@groeges your bug has been fixed in 0.5, and file:// URL support has been added. We are just starting testing now but I can let you know when we publish.
